### PR TITLE
Add LongestCommonPrefix.java

### PR DESCRIPTION
**Pull request by TheDarkW3b**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.
- [x] All new code is formatted with `clang-format -i --style=file path/to/your/file.java`

### CODE REVIEW

File: src/main/java/com/thealgorithms/strings/LongestCommonPrefix.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.thealgorithms.strings;
+
+import java.util.Arrays;
+
+public final class LongestCommonPrefix {
```
Comment (saahil-mahato) [inline src/main/java/com/thealgorithms/strings/LongestCommonPrefix.java:5]: Please add tests.

File: src/main/java/com/thealgorithms/strings/LongestCommonPrefix.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    public String longestCommonPrefix(String[] strs) {
+        if(strs == null || strs.length == 0) return "";
```
Comment (saahil-mahato) [inline src/main/java/com/thealgorithms/strings/LongestCommonPrefix.java:7]: Please follow styling standard. `if` block should be inside `{}`.

File: src/main/java/com/thealgorithms/strings/LongestCommonPrefix.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+        Arrays.sort(strs);
+        String shortest = strs[0];
+        String longest = strs[strs.length - 1];
+
+        int index = 0;
+        while(index < shortest.length() && index < longest.length() && shortest.charAt(index) == longest.charAt(index))
```
Comment (saahil-mahato) [inline src/main/java/com/thealgorithms/strings/LongestCommonPrefix.java:14]: Please follow styling standard. `while` block should also be inside `{}`.

**Review by saahil-mahato — CHANGES_REQUESTED** [review]

**Comment by TheDarkW3b** [comment]
Let me know if any changed are required.. also please add the hacktoberfest label

**Comment by TheDarkW3b** [comment]
Let me know if any changes are required.. also please add the hacktoberfest label

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Pull request merged**